TrafficCapture/dockerSolution/src/main/docker/migrationConsole/lib/cluster_curl/Pipfile

```diff
@@ -0,0 +1,14 @@
+[[source]]
```

Comment:
Why not use reuse the same workflow we have in clusters tool which uses `cluster.call_api(...)`?

Reply:
Oh, I missed the usage case at the top or your description, my apologyizes.

I was expecting our users to be able to run `console cluster curl target -XGET /foo/bar` - any reason why not? Seems like starting with `console ...` is a way to organize and group the MA functionality.